javascript/javascript3/week1/Weather App/main.js

```diff
@@ -0,0 +1,77 @@
+let inputValue = document.querySelector(".inputValue");
```

Comment:
The trouble with `let` variables is, that they can be re-defined later in your code. If you don't want to re-define your variable, use `const` instead. It makes sure you can't accidentally re-define your variable 🔒

Reply:
Oh Thank you for explaining this point, I fix it